mod/calculated_metrics/set/type/metric/calculations.rb

```diff
@@ -10,7 +10,9 @@
 end
 
 def calculation_in_progress!
-  Answer.where(id: all_dependent_answer_ids).update_all(calculating: true)
+  ids = all_dependent_answer_ids
+  Answer.where(id: ids).update_all(calculating: true)
+  Answer.where(id: ids).each(&:expire)
```

Comment:
Aren't these answer objects and not cards?

Reply:
Weird, Answer objects have an `expire` method, but I can't figure out where it's defined.

```
[5] pry(main)> Answer.first.method(:expire).source_location
=> nil
```

Reply:
Answer objects delegate missing method calls to the answer card object.

Reply:
Ahhhh.  thank you.